src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(String color, int firstSide, int secondSide) {
+        this.color = color;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, color: " + getColor() + ", area: " + getArea()
+                + " sq. units, first side: " + firstSide + " units, second side: "
+                + secondSide + " units");
+    }
```

Comment:
Your `Rectangle` class currently extends `Figure` class which is fine as long as `Figure` is not an abstract class that is used to set behavior. Make sure that `Figure` is either a concrete class providing common fields and methods for its subclasses or an interface that `Rectangle` implements if it only provides method signatures.